File: Excel_Programs/Python/question11.py
n=int(input("Enter the number of elements to be entered : "))
arr=[]
for i in range(n):
    arr.append(int(input("Enter a number : ")))
x=0
re=0

for i in range(len(arr)):
    c=0
    for j in range(len(arr)):
        if arr[i] == arr[j]:
            c+=1
    if(c>1):
        x=c
        re=arr[i]
print(re," ",x)

File: Excel_Programs/C++/question11.cpp
#include<iostream>
using namespace std;
 int main()
 {
     int n;
     cout<<"Enter the number of elements to be entered : ";
     cin>>n;
     int arr[n];
     for(int i =0;i<n;i++)
     {
        cout<<"Enter a number : ";
        cin>>arr[i];
     }
     cout<<"\n";
     int x=0,re=0;
        for(int i = 0; i<n;i++)
        {
            int c=0;
            for(int j = 0;j<n;j++)
            {
                if(arr[i]==arr[j])
                {
                    c++;
                }
            }
            if(c>1)
            {
                x=c;
                re=arr[i];
            }
        }
        cout<<"\n"<<re<<"  "<<x<<"\n";
 }